engine/src/main/java/org/hibernate/search/engine/cfg/spi/ParseUtils.java

```diff
@@ -366,4 +367,32 @@ public static <T> T parseDiscreteValues(T[] allowedValues, Function<T, String> s
 						.collect( Collectors.toList() )
 		);
 	}
+
+	public static byte[] parseBytePrimitiveArray(String value) {
+		try {
+			String[] values = value.replaceAll( "(^\\s*\\[\\s*)|(\\s*\\]\\s*$)", "" ).split( "[,;\\s]+" );
```

Comment:
Sonar is warning about DOS attacks here. We'll need to address this in a follow-up issue.

Reply:
https://hibernate.atlassian.net/browse/HSEARCH-5023